src/views/BookListView.tsx

```diff
@@ -55,6 +72,12 @@ const BookListView = () => {
           <div className="flex justify-center italic">No results</div>
```

Comment:
Can be replaced with the newly-added NoResults component, like this:
```suggestion
          <NoResults />
```